roles/postgres/templates/postgres-backups@.service.j2

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description = Back up the PostgresSQL database `%i`
+Requires = postgresql.service
+After = postgresql.service
+
+[Service]
+User = postgres
+Group = postgres
+WorkingDirectory = {{ postgres_backups_dir }}
+ExecStartPre = +/usr/bin/chown postgres:postgres /var/lib/postgresql-backups
```

Comment:
`/var/lib/postgresql-backups` -> `{{ postgres_backups_dir }}`

Reply:
Oops, missed this one!